yt_dlp/extractor/tvw.py

```diff
@@ -10,12 +10,12 @@
     unified_timestamp,
     url_or_none,
 )
-from ..utils.traversal import find_element, traverse_obj
+from ..utils.traversal import find_element, find_elements, traverse_obj
 
 
 class TvwIE(InfoExtractor):
     IE_NAME = 'tvw'
-    _VALID_URL = r'https?://(?:www\.)?tvw\.org/video/(?P<id>[^/?#]+)'
+    _VALID_URL = r'https?://(?:www\.)?tvw\.org/(?:video|watch)/?(?:\?eventID=)?(?P<id>[^/?#]+)'
```

Comment:
that pattern would allow for `https://tvw.org/watchID123123` etc

I think we should do this:
```suggestion
    _VALID_URL = [
        r'https?://(?:www\.)?tvw\.org/video/(?P<id>[^/?#]+)',
        r'https?://(?:www\.)?tvw\.org/watch/?\?(?:[^#]+&)?eventID=(?P<id>\d+)',
    ]
```